src/styles/inherited-scss/helpers/_icons.scss

```diff
@@ -0,0 +1,141 @@
+@use "sass:math";
```

Comment:
Caused by `npx sass-migrator division **/*.scss` (was needed to silence a warning)